src/pages/instances/EditInstance.tsx

```diff
@@ -112,13 +113,20 @@ const EditInstance: FC<Props> = ({ instance }) => {
       instancePayload.etag = instance.etag;
 
       void updateInstance(instancePayload, project).then((operation) => {
+        const instanceLink = generateInstanceLink({ operation, project });
+        if (!instanceLink) return;
         eventQueue.set(
           operation.metadata.id,
           () => {
-            notify.success("Instance updated.");
+            toastNotify.success(<>Instance {instanceLink} updated.</>);
             void formik.setValues(getInstanceEditValues(instancePayload));
           },
-          (msg) => notify.failure("Instance update failed", new Error(msg)),
+          (msg) =>
+            toastNotify.failure(
+              "Error",
+              new Error(msg),
+              <>Instance {instanceLink} update failed.</>,
+            ),
```

Comment:
Error title?

Reply:
In this case, the title can't be a react node and I wanted to have the `instanceLink` in the notification. I think I will have `Instance update failed.` in the title then just keep the link to the instance in the message body.

Reply:
I've done this for all generic `Error` titles